src/index.html

```diff
@@ -11,7 +11,126 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
```

Comment:
This `<script>` tag should be after all tags inside `<body>`:
```
<body>
  <!-- Your content -->
  
  <script type="text/javascript" src="./main.js"></script>
</body>
```

P.s.: here material about this: https://uk.javascript.info/script-async-defer